docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/deploy-cluster.md

```diff
@@ -28,12 +28,12 @@ dpkg -i nebula-{VERSION}.{SYSTEM_VERSION}.amd64.deb
 
 ### 启动 Nebula Graph 集群
 
-目前，**Nebula Graph** 集群由 `scripts/services.sh` 运维，可使用此脚本 `start`，`stop` 或 `restart` 重启集群。
+目前，**Nebula Graph** 集群由 `scripts/nebula.service.` 运维，可使用此脚本 `start`，`stop` 或 `restart` 重启集群。
```

Comment:
can't directly modify `services.sh` to `nebula.service`, this doc is about `how to deploy cluster`, you should modify the whole doc

Reply:
I'll take care of this later since we will change the way to deploy.